dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/Index.cshtml

```diff
@@ -0,0 +1,84 @@
+        <govuk-summary-list>
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Name</govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>@Model.Name</govuk-summary-list-row-value>
+                <govuk-summary-list-row-actions>
+                    <govuk-summary-list-row-action href="#" visually-hidden-text="name">Change</govuk-summary-list-row-action>
+                </govuk-summary-list-row-actions>
+            </govuk-summary-list-row>
+            @if (Model.HasTrn)
+            {
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>Official name</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>
+                        @Model.OfficialName
+                        <p class="govuk-hint govuk-!-font-size-14">Displayed on teaching certificates </p>
+                    </govuk-summary-list-row-value>
+                    <govuk-summary-list-row-actions>
+                        <govuk-summary-list-row-action href="#" visually-hidden-text="official name">Change</govuk-summary-list-row-action>
+                    </govuk-summary-list-row-actions>
+                </govuk-summary-list-row>
+            }
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Date of birth</govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>@Model.DateOfBirth?.ToString("dd MMMM yyyy")</govuk-summary-list-row-value>
+                <govuk-summary-list-row-actions>
+                    <govuk-summary-list-row-action href="#" visually-hidden-text="date of birth">Change</govuk-summary-list-row-action>
+                </govuk-summary-list-row-actions>
+            </govuk-summary-list-row>
+        </govuk-summary-list>
+
+        <h3 class="govuk-heading-m">Sign in details</h3>
+        <govuk-summary-list>
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Email</govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>@Model.Email</govuk-summary-list-row-value>
+                <govuk-summary-list-row-actions>
+                    <govuk-summary-list-row-action href="#" visually-hidden-text="email">Change</govuk-summary-list-row-action>
+                </govuk-summary-list-row-actions>
+            </govuk-summary-list-row>
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Mobile number</govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>@Model.MobileNumber</govuk-summary-list-row-value>
+                <govuk-summary-list-row-actions>
+                    <govuk-summary-list-row-action href="#" visually-hidden-text="mobile number">Change</govuk-summary-list-row-action>
+                </govuk-summary-list-row-actions>
+            </govuk-summary-list-row>
+        </govuk-summary-list>
+
+        @if (Model.SafeRedirectUri is not null && Model.ClientDisplayName is not null)
```

Comment:
Is there a reason the logic of this back button is different to the one at the top of the page?

Reply:
It was a hangover of making this page reachable from the `/Complete` page. Since we don't need that quite yet, I've aligned the conditions for both.